File: BackEnd/src/main/java/com/Checkerly/BackEnd/resources/exception/StandardError.java
package com.Checkerly.BackEnd.resources.exception;

import java.io.Serial;
import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor

public class StandardError implements Serializable{
	@Serial
	private static final long serialVersionUID = 1L;
	
	private Long timestamp;
	private Integer status;
	private String error;
	private String message;
	private String path;
}

File: BackEnd/src/main/kotlin/com/Checkerly/BackEnd/resources/exception/StandardError.kt
package com.Checkerly.BackEnd.resources.exception

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import java.io.Serializable

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
class StandardError : Serializable {
    private val timestamp: Long? = null
    private val status: Integer? = null
    private val error: String? = null
    private val message: String? = null
    private val path: String? = null

    companion object {
        private const val serialVersionUID: Long = 1L
    }
}
